### DISCUSSION TITLE
import device - ip ?

### DISCUSSION THREAD

**Discussion by y0d4a** [netbox-community/netbox#17958] [Help Wanted!]
Hi, i need to import list of devices with their IP, in import i do not see field IP address and cannot import it?
but in the list there is devices with IP addresses...

path for import: netbox/dcim/devices/import/
fields which i have:
<img width="1447" alt="image" src="https://github.com/user-attachments/assets/e35dd634-d4d9-44ed-8ca4-c18a362bdb5c">

field which i need:
<img width="2382" alt="image" src="https://github.com/user-attachments/assets/32c06846-012d-447c-876d-2a495a505400">

**Comment by zmaster7** [comment]
IP addresses are a separate model from devices, and would have to be imported separately from devices, under: ipam/ip-addresses/import/

The addresses are assigned to an interface on the device, and can be specified as the primary ip for that device (is_primary).

![image](https://github.com/user-attachments/assets/c1e38bf3-d29a-45cb-a7c3-bc91aba0ab71)

**Comment by y0d4a** [comment]
ah, understand, thank you for clarification, i succeeded.
cheers!
